core_backend/app/contents/routers.py

```diff
@@ -58,7 +63,9 @@ async def create_content(
 ) -> Optional[ContentRetrieve]:
     """
     Create content endpoint. Calls embedding model to get content embedding and
-    inserts it to PG database
+    inserts it to the content table in the database.
+
+    ⚠️ To add tags, use the tags endpoint to create tags first.
```

Comment:
These docstrings are rendered on Swagger UI as endpoint description, so made improvements here and there!